signatures.md

```diff
@@ -123,93 +96,3 @@ sig SignatureBytes = [0x30][len][0x02][r][indicator][s][indicator][recovery]
 
 From: <https://github.com/bitcoin-core/secp256k1/blob/314a61d72474aa29ff4afba8472553ad91d88e9d/src/ecdsa_impl.h#L177>
 
-**Signed Message**
-
-```
-Type SignedMessage Struct {
-
-    Message bytes `JSON`   
-
-    Signature bytes `JSON` 
-    
-}
-```
-
-[Current Filecoin Implementation](https://github.com/filecoin-project/go-filecoin/blob/ab60f73af1f86a954ef41e3252ef52a99066bbe2/types/signed_message.go#L25)
-
-
-### Github
-
-<https://github.com/filecoin-project/specs/issues/131>
-
-### External References
```

Comment:
as mentioned in person, please keep the references to the specs and rfcs either here or in a general "Reference" section